### PR TITLE
Dockerize app#11334

### PR DESCRIPTION
**Pull request by jakubzimnol**

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.6
+
+ENV BACKEND_PATH /backend
+
+WORKDIR ${BACKEND_PATH}
+
+RUN apt-get update -q && apt-get upgrade -qy
```
Comment (Marcin-Josinski) [inline backend/Dockerfile]: Why do you need update repositories?

Comment (marcinjosinski) [inline backend/Dockerfile]: Why do you need to update repositories?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  backend:
+    build: ./backend
+    depends_on:
+      - db
+    ports:
+      - ${PORTS}
+    env_file:
+      - .env
+    container_name: backend
+    command: bash -c "export FLASK_ENV=development
```
Comment (marcinjosinski) [inline docker-compose.yml]: Move `FLASK_ENV`variable to .env file.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    volumes:
+      - ./backend:/backend
+
+  db:
+    image: postgres
+    env_file:
+      - .env
+
+volumes:
+  static-files:
```
Comment (Marcin-Josinski) [inline docker-compose.yml]: Why do you need this? 😄

Comment (marcinjosinski) [inline docker-compose.yml:34]: Why do you need this volume? 😄

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on:
+      - db
+    ports:
+      - ${PORTS}
+    env_file:
+      - .env
+    container_name: backend
+    command: bash -c "export FLASK_ENV=development
+      && export FLASK_APP=app.py
+      && flask run --host 0.0.0.0 --port 8000"
```
Comment (marcinjosinski) [inline docker-compose.yml]: I think that port 8000 is set by default.

File: docker-compose.yml (hunk 3 above)
Comment (marcinjosinski) [inline docker-compose.yml:34]: Where is docker for production?

File: docker-compose.yml (hunk 4 above)
Comment (jakubzimnol) [reply docker-compose.yml]: flask sets port 5000 by default

File: docker-compose.yml (hunk 3 above)
Comment (jakubzimnol) [reply docker-compose.yml:34]: I haven't have nginx yet, so I think it's not necessary to have docker for production

**Review by marcinjosinski — APPROVED** [review]

File: backend/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+ENV BACKEND_PATH /backend
+
+WORKDIR ${BACKEND_PATH}
+
+COPY requirements.txt ${BACKEND_PATH}
+
+RUN pip install -r requirements.txt
+
+COPY . ${BACKEND_PATH}
```
Comment (grzegorz-bielski) [inline backend/Dockerfile]: Aren't you already in `BACKEND_PATH` ? Wouldn't `COPY . .` work just fine?

File: docker-compose.yml (hunk 3 above)
Comment (grzegorz-bielski) [reply docker-compose.yml:34]: It'd be easier for you to have a full dockerized setup from the start

**Review by grzegorz-bielski — APPROVED** [review]

**Pull request merged**